_posts/2015-03-31-how-we-do-css.md

```diff
@@ -0,0 +1,173 @@
+
+We support the last few versions of popular browsers (Chrome, Safari and Firefox). At the time of writing this we currently support IE 8+. About 2% of our users still use IE 8; we are not out of the woods yet.
+
+We made the leap to a responsive application with our recent redesign of the site. Therefore, we support a wide range of desktop, tablet and mobile devices. At the time of writing this we don’t currently support smart watches. We will soon, though… so, "watch" out for that... sorry.
+
+
+### Preprocessor
+We use Sass with Compass as our preprocessor. Our team is proficient and has much training on this popular pre-processor and it’s large following helps.
+
+
+### Folder/Sandwich Structure
+Here’s the fun stuff - we structured our css based on a few important factors:
+
+
+#### + Modularity and Reusability
+Develop components that can be reused throughout the application.
+
+
+#### + CSS Rules
+Consider the size of the compiled stylesheets per section of the site to not exceed IE9’s rules limit of 4,095. Our previous site was compiled into two massive stylesheets and we learned the hard way. We currently break it down per section of the site. See examples below.
+
+
+#### + Light stylesheets
+Keeping the stylesheet declaration low kills two birds with one stone. We break the stylesheets down into individual sheets that contain only the rules for each section of the site—this has the added benefit of avoiding unnecessary or redundant styles per section. Doing this helps us keep our stylesheets light.
+
+Can I have my sandwich with a side of css?
+Zumba is a happy and energetic fitness lifestyle company. That doesn’t mean we can’t have our sandwich and eat it too. We decided to structure our css like a sandwich order.
+
+<div><img src="/img/blog/sandwich-breakdown.jpg" alt="sandwich" class="img-responsive"/></div>
+
+
+#### Plate
+The plate represents the foundation of the code and is reused throughout the site.
+Example:
+
+    plate
+    |— _angular.scss
+    |— _animations.scss
+    |— _fonts.scss
+    |— _icons.scss
+    |— _mixins.scss
+    |— _typography.scss
+    |— _variables.scss
+
+
+#### Toppings
+These can be added to your sandwich to make it as delicious as you want it. These consist of mini components that can be reused throughout the site, but is not necessary for every page.
+Example:
+
+	toppings
+	|— _button.scss
+	|— _input.scss
+	|— _shares.scss
+	|— _tooltip.scss
+
+
+#### Sandwich
+These nom noms can contain a combination of toppings to make a complete sandwich or component. These are usually a bit more complex and a little more specific. Sandwiches can be used throughout the entire site and can be customized per section of the site to meet that section's needs or requirements.
```

Comment:
Sandwiches can be used throughout the entire site<strong>,</strong> and can be customized per section <strike>of the site</strike> to meet that section's needs or requirements.